Libplanet.Extensions.Cocona/Commands/StoreQueryCommand.cs

```diff
@@ -0,0 +1,135 @@
+using System;
+using System.Collections.Immutable;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using Bencodex.Types;
+using Cocona;
+using Libplanet.Action;
+using Libplanet.Blocks;
+using Libplanet.Tx;
+
+namespace Libplanet.Extensions.Cocona.Commands
+{
+    public class StoreQueryCommand
+    {
+        [Command(Description = "Query block by the index")]
```

Comment:
For consistency, CLI descriptions should be:

- Start with a capital letter.
- End with a period.
- Prescribe the function as a command (“do”) rather than a description (“does”) if a sentence starts with a verb.
- An article shouldn't be omitted.

Of course, these are just for style, not rules.

```suggestion
        [Command(Description = "Query a block by index.")]
```

Reply:
Applied. Thanks.